.circleci/config.yml

```diff
@@ -122,7 +125,23 @@ jobs:
           dist_tag: << parameters.dist_tag >>
           version_bump: << parameters.version_bump >>
 
-  job_pr-validator:
+  job_pr-validator-win32:
+    environment:
+      NODE_OPTIONS: --max-old-space-size=14336
+      CI: ""
+    parameters:
+      shell:
+        type: string
+    executor:
+      name: win/default
+      shell: << parameters.shell >>
+      size: xlarge
+    steps:
+      - checkout
+      - cmd_install-dependencies
+      - cmd_run-unit-tests
```

Comment:
I am hoping we want to only lint once and there are no difference in output of linter between node and win32

Reply:
Linting would fail on windows because of `newline` being different. Linting should only run on the OS that the developers develop on anyways.